### ISSUE TITLE
SeaTunnel Community Meeting

### ISSUE THREAD

**Issue by hailin0** [apache/seatunnel#6457]
### Search before asking

- [X] I had searched in the [feature](https://github.com/apache/seatunnel/issues?q=is%3Aissue+label%3A%22Feature%22) and found no similar feature requirement.

### Description

hailin0 invites you to a meeting on Tencent Meeting
Meeting Topic: SeaTunnel Community Meeting
Time: 2024/03/12 20:00-21:00 (GMT+08:00) China Standard Time - Beijing

Click the link to join the meeting or to add it to your meeting list:
https://meeting.tencent.com/dm/XpbjTe31JOSc

#TencentMeeting: 691-201-146

Copy this invitation and open in Tencent Meeting app (V2.13 or later) to join the meeting
Everyone is very welcome to join:
If you have an issue, please submit it one day before the meeting, and provide a specific issue link and context.

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by hailin0** [comment]
@hailin0 
1. 2.3.4 Release(https://github.com/apache/seatunnel/releases/tag/2.3.4)
2. new type converter(https://github.com/apache/seatunnel/pull/5872)
3. event listener (https://github.com/apache/seatunnel/pull/6419)

**Comment by hailin0** [comment]
@EricJoy2048
2024 roadmap

**Comment by gitfortian** [comment]
is zeta support write and read kafka with kerberos ,how to set the conf

**Comment by SimonChou12138** [comment]
Whether to consider allowing paimon to support concurrent writes? At present, the Writer will write to different buckets at the same time, which will cause write conflicts during the final commit.
In fact, in the official Paimon Api, it is possible to detect which Bucket the Row is in, build a Writer for the corresponding bucket, and then write to it, so that there will be no write conflicts at the end of the commit.
In fact, the `write-only` parameter of the Paimon table can be used to solve the write conflict problem, but it will have some drawbacks, such as too many small files, skipping compression and deleting expired data.
I have tried that the java api can be implemented, and I have tried to modify it in the existing Paimon Connector. The test in the example is normal, but there are still problems in testing actual tables in production environment. It is considered that the modified write process cannot completely avoid this problem, and it has to be calculated and implemented at the upper level of the flow
